src/Compilers/CSharp/Test/Emit/Emit/EditAndContinue/EditAndContinueTests.cs

```diff
@@ -1936,6 +1936,202 @@ public C()
+                    new SemanticEdit(SemanticEditKind.Insert, null, y1),
+                    new SemanticEdit(SemanticEditKind.Insert, null, m1),
+                    new SemanticEdit(SemanticEditKind.Insert, null, n1),
+                    new SemanticEdit(SemanticEditKind.Insert, null, p1),
+                    new SemanticEdit(SemanticEditKind.Insert, null, q1),
+                    new SemanticEdit(SemanticEditKind.Insert, null, e1),
+                    new SemanticEdit(SemanticEditKind.Insert, null, f1),
+                    new SemanticEdit(SemanticEditKind.Insert, null, j1),
+                    new SemanticEdit(SemanticEditKind.Insert, null, cctor1)));
+
+            using var md1 = diff1.GetMetadata();
+            var reader1 = md1.Reader;
+            var readers = new[] { reader0, reader1 };
+
+            CheckNames(readers, reader1.GetTypeDefNames(), "J");
+            CheckNames(readers, reader1.GetFieldDefNames(), "X", "Y");
+            CheckNames(readers, reader1.GetMethodDefNames(), "add_Y", "remove_Y", "M", "N", "get_P", "set_P", "get_Q", "set_Q", "add_E", "remove_E", "add_F", "remove_F", ".cctor");
+            Assert.Equal(1, reader1.GetTableRowCount(TableIndex.NestedClass));
+
+            var diff2 = compilation2.EmitDifference(
+                diff1.NextGeneration,
+                ImmutableArray.Create(
+                    new SemanticEdit(SemanticEditKind.Update, x1, x2),
+                    new SemanticEdit(SemanticEditKind.Update, m1, m2),
+                    new SemanticEdit(SemanticEditKind.Update, n1, n2),
+                    new SemanticEdit(SemanticEditKind.Update, getP1, getP2),
+                    new SemanticEdit(SemanticEditKind.Update, setP1, setP2),
+                    new SemanticEdit(SemanticEditKind.Update, getQ1, getQ2),
+                    new SemanticEdit(SemanticEditKind.Update, setQ1, setQ2),
+                    new SemanticEdit(SemanticEditKind.Update, addE1, addE2),
+                    new SemanticEdit(SemanticEditKind.Update, removeE1, removeE2),
+                    new SemanticEdit(SemanticEditKind.Update, addF1, addF2),
+                    new SemanticEdit(SemanticEditKind.Update, removeF1, removeF2),
+                    new SemanticEdit(SemanticEditKind.Update, cctor1, cctor2)));
+
+            using var md2 = diff2.GetMetadata();
+            var reader2 = md2.Reader;
+            readers = new[] { reader0, reader1, reader2 };
+
+            CheckNames(readers, reader2.GetTypeDefNames());
+            CheckNames(readers, reader2.GetFieldDefNames(), "X");
+            CheckNames(readers, reader2.GetMethodDefNames(), "M", "N", "get_P", "set_P", "get_Q", "set_Q", "add_E", "remove_E", "add_F", "remove_F", ".cctor");
+            Assert.Equal(0, reader2.GetTableRowCount(TableIndex.NestedClass));
+
+            CheckEncLog(reader2,
+                Row(3, TableIndex.AssemblyRef, EditAndContinueOperation.Default),
```

Comment:
>Row(3, [](start = 16, length = 6)

I don't know how to interpret these numbers. Are they important?

Reply:
These are row ids. Each line here corresponds to an entry in EnC log that says what operation was done on what entity (row number + table id). The specific values are not that interesting. The interesting part is that we are emitting a couple of events, a field, a bunch of MethodDefs and and a couple of properties.